package/wolfssl/wolfssl.mk

```diff
@@ -0,0 +1,50 @@
+################################################################################
+#
+# wolfssl
+#
+################################################################################
+
+WOLFSSL_VERSION = 5.5.0-stable
+WOLFSSL_SITE = $(call github,wolfSSL,wolfssl,v$(WOLFSSL_VERSION))
+WOLFSSL_INSTALL_STAGING = YES
+
+WOLFSSL_LICENSE = GPL-2.0+
+WOLFSSL_LICENSE_FILES = COPYING LICENSING
+WOLFSSL_CPE_ID_VENDOR = wolfssl
+
+WOLFSSL_DEPENDENCIES = host-pkgconf
+
+# wolfssl's source code is released without a configure
+# script, so we need autoreconf
+WOLFSSL_AUTORECONF = YES
+
+WOLFSSL_CONF_OPTS = --disable-examples --disable-crypttests --enable-opensslall --enable-keygen --enable-rsapss --enable-des3 --enable-dtls \
+ --enable-certgen --enable-certreq --enable-certext --enable-sessioncerts --enable-crl --enable-ocsp --enable-static
```

Comment:
Would `--enable-pem` and `--enable-pki` also be necessary here?

Reply:
Hi, 
according to this doc: https://www.wolfssl.com/strongswan-wolfssl-fips-3/
they should be added to strongswan

Reply:
Makes sense.